packages/lib/multiwallet/multiwallet-ethereum-mewconnect-connector/src/index.ts

```diff
@@ -28,6 +28,12 @@ export class EthereumMEWConnectConnector extends AbstractEthereumConnector<MewPr
     constructor(options: EthereumConnectorOptions) {
         super(options);
         this.chainId = options.chainId;
+        for (let rpc of Object.values(options.rpc)) {
+            console.log(rpc);
+            if (rpc[0] !== "w") {
```

Comment:
Should this check the entire "wss://" or "ws://" string? (e.g. in case the rpc is missing the protocol entirely)

Reply:
The MewConnect client will throw if the rpc is invalid (it "works" with an http/s rpc endpoint, but fails silently later). Will remove the console call though